rdr_service/genomic/genomic_job_components.py

```diff
@@ -952,14 +953,22 @@ def ingest_metrics_file_from_filepath(self, metric_type, file_path):
 
             if item_count == batch_size:
                 with metric_dao.session() as session:
-                    session.bulk_save_objects(batch)
+                    # Use session add_all() so we can get the newly created primary key id values back.
```

Comment:
@j-kanuch and @robabram :  How many rows are expected to be impacted here and are there negative implications to changing from bulk operation to `add_all()`?  What was the motivation for using the bulk operation originally?

Reply:
Resolved offline, Robert relayed this was already discussed with Josh and it should not impact performance/load.

Reply:
@pegbertsch The `batch_size` is set to 100. The `bulk_save_objects()` method was used because we only needed to insert and didn't need any of the object data. The `add_all()` method returns the object data, and because this is needed for PDR, this is now being used. The `bulk_save_objects` is more performative, but since these objects are not very big, the performance cost of using `add_all` shouldn't be too noticeable. We probably could have used `add_all` to begin with for this object.